app/res/values/attrs.xml

```diff
@@ -49,4 +49,9 @@
         <attr name="clip_height" format="reference|float"/>
     </declare-styleable>
 
+    <declare-styleable name="FilePreference">
+        <attr name="file_type" format="string"/>
+        <attr name="error_dialog_title" format="string"/>

```

Comment:
Hey, so while this seems like a clean solution for customizing the dialog title, I'm still not sure why it's necessary, and this seems like a lot of setup for something that we could just not do. Was there a specific reason you saw that made it seem more important to customize the title, as opposed to just going with something like "No File Browser Available"?